android/quest/src/main/res/navigation/application_nav_graph.xml

```diff
@@ -3,7 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/nav_graph"
-    app:startDestination="@id/registerFragment">
+    app:startDestination="@id/geoWidgetLauncherFragment">
```

Comment:
Revert to registerFragment. Then use a navigation argument to switch between the map and register views. We can default to register view. The nav argument can be of `enum` type. 


```kotlin
enum class Launcher {
  MAP,
  REGISTER
}
```

Reply:
Updated it on https://github.com/opensrp/fhircore/pull/3148